docker/Dockerfile

```diff
@@ -1,44 +1,42 @@
 FROM debian:buster AS stage_build
 
-ARG EMSCRIPTEN_VERSION=1.39.18
+ARG EMSCRIPTEN_VERSION=1.39.20
 ENV EMSDK /emsdk
 
 # ------------------------------------------------------------------------------
 
 RUN echo "## Start building" \
-    \
-&&	echo "## Update and install packages" \
-    &&	apt-get -qq -y update \
-    &&  apt-get -qq install -y --no-install-recommends \
-            libxml2 \
-            wget \
-            git-core \
-            ca-certificates \
-            build-essential \
-            file \
-            python3 python3-pip \
-&&  echo "## Done"
-
-RUN  echo "## Get EMSDK" \
-    &&  git clone https://github.com/emscripten-core/emsdk.git ${EMSDK} \
-&&  echo "## Done"
-
-RUN  echo "## Install Emscripten" \
-    &&  cd ${EMSDK} \
-    &&  ./emsdk install ${EMSCRIPTEN_VERSION} \
-&&  echo "## Done"
+    && echo "## Update and install packages" \
+    && apt-get -qq -y update \
+    && apt-get -qq install -y --no-install-recommends \
+        binutils \
+        build-essential \
+        ca-certificates \
+        file \
+        git \
+        python3 \
+        python3-pip \
+    && echo "## Done"
+
+RUN echo "## Get EMSDK" \
+    && git clone https://github.com/emscripten-core/emsdk.git ${EMSDK} \
+    && echo "## Done"
+
+RUN echo "## Install Emscripten" \
+    && cd ${EMSDK} \
+    && ./emsdk install ${EMSCRIPTEN_VERSION} \
+    && echo "## Done"
 
 # This generates configuration that contains all valid paths according to installed SDK
 RUN cd ${EMSDK} \
-    &&  echo "## Generate standard configuration" \
-    &&  ./emsdk activate ${EMSCRIPTEN_VERSION} \
-    &&  chmod -R 777 ${EMSDK}/upstream/emscripten/cache \
-&&  echo "## Done"
-
-# Clean up emscripten installation and strip some symbols
-RUN echo "## Aggresive optimization: Remove debug symbols" \
-&&  apt-get -qq -y update && apt-get -qq install -y --no-install-recommends \
-        binutils \
+    && echo "## Generate standard configuration" \
+    && ./emsdk activate ${EMSCRIPTEN_VERSION} \
+    && chmod -R 777 ${EMSDK}/upstream/emscripten/cache \
+    && cat ${EMSDK}/upstream/emscripten/cache/sanity.txt \
```

Comment:
I think it was a bug that we ever shipped a copy of the sanity file.   This was fixed recently: https://github.com/WebAssembly/waterfall/pull/656.

In the past (a few weeks ago) the sanity file was not part of the cache and so didn't get shipped.   Now that it is we accidentally started shipping it. 

This means the cache cleared on first use (because the paths in the sanity file are don't match the installation).

For now the best thing to do is remove this sanity file and instead have it generated by the block below. 

Its not ideal but its the the logic works today:

1. Bad sanity file = clear cache
2. No sanity file = leave catch alone and generate sanity file.

Reply:
fwiw, I still see this behavior with the tip-of-tree build, see:

```bash
$ ./emsdk install tot-upstream
...
Done installing SDK 'sdk-releases-upstream-1ba7fc82899ab241d0322d5ab20e27ec50cbbf9b-64bit'.
$ [ -e "upstream/emscripten/cache/sanity.txt" ] && echo "sanity.txt exists" || echo "sanity.txt not found"
sanity.txt not found
$ ./emsdk activate tot-upstream
Setting the following tools as active:
   node-12.18.1-64bit
   releases-upstream-1ba7fc82899ab241d0322d5ab20e27ec50cbbf9b-64bit
...
$ [ -e "upstream/emscripten/cache/sanity.txt" ] && echo "sanity.txt exists" || echo "sanity.txt not found"
sanity.txt exists
```

Reply:
Great.  There was just a short period of time where we were accidentally shipping the sanity file.  Hopefully that is fixed now and we are good to go.

As long as there is no sanity file generation of cache file population happening when the image gets used that is the main thing.